docs/cli-arguments.md

```diff
@@ -14,6 +14,7 @@
 | `--apiserver-host` | `string` | The Kubernetes API server URL. If not set, the controller will use cluster config discovery. |  |
 | `--apiserver-qps` | `int` | The Kubernetes API RateLimiter maximum queries per second. | `100` |
 | `--cache-sync-timeout` | `duration` | The time limit set to wait for syncing controllers' caches. Set to 0 to use default from controller-runtime. | `2m0s` |
+| `--cluster-domain` | `string` | The cluster domain. This is used e.g. in generating addresses for upstream services. | `cluster.local` |
```

Comment:
```suggestion
| `--cluster-domain` | `string` | The cluster domain. This is used e.g. in generating addresses for upstream services. | `` |
```

Reply:
Forgot to generate CLI args. 74ff3be8d230e62c439d77367ffa47a59cd6618a does that.

FYI we generate an empty string rather then verbatim code tags to contain an empty string.